app/views/posts/index.html.erb

```diff
@@ -0,0 +1,28 @@
+<%= stylesheet_link_tag('index') %>
+<div class="container">
+  <div class="user">
+    <img src="<%= image_path('example.com.png') %>" alt="image">
+    <div class= "text-container">
+      <h1>Username</h1>
+      <p class= "inner-text">Number of posts x</p>
+    </div>
+  </div>
+  <div>
+    <h2>post 1</h2>
+    <p>comment 1</p>
+    <p>comment 2</p>
+    <p>comment 3</p>
+  </div>
+  <div>
+    <h2>post 2</h2>
+    <p>comment 1</p>
+    <p>comment 2</p>
+    <p>comment 3</p>
+  </div>
+  <div>
+    <h2>post 3</h2>
+    <p>comment 1</p>
+    <p>comment 2</p>
```

Comment:
- You have done well setting up your view file for the `post#index action`, but according to the project requirements, you are to set up a `posts` controller with the index action, please fix this 😄 
**OPTIONAL** - Also, please note that you are not required to have any logic in your view files for this milestone, your view file is allowed to have just a placeholder. Please fix this 👍 👍